### PR TITLE
fix: redraw after toggling on `CmdlineEnter` event

### PR DESCRIPTION
**Pull request by adam-ja**
Redraw the screen after toggling on the CmdlineEnter event, otherwise the change does not appear to take effect for this event only.

Credit to https://vi.stackexchange.com/a/31218

### CODE REVIEW

File: lua/relative-toggle/init.lua [hunk 1]
```diff
@@ -15,11 +15,15 @@ local function create_autocmd(event, opts)
...
 ---@param relative boolean #whether relativenumber should be set
-local function set_relativenumber(relative)
+---@param redraw boolean #whether to redraw the screen
+local function set_relativenumber(relative, redraw)
     local in_insert_mode = vim.api.nvim_get_mode().mode == "i"
 
     if vim.o.number then
         vim.opt.relativenumber = relative and not in_insert_mode
+        if redraw then
+            vim.cmd("redraw")
```
Comment (adam-ja) [inline lua/relative-toggle/init.lua:25]: I couldn't find a lua API equivalent for this 😞

File: lua/relative-toggle/init.lua [hunk 2]
```diff
@@ -15,11 +15,15 @@ local function create_autocmd(event, opts)
...
 
 ---@param relative boolean #whether relativenumber should be set
-local function set_relativenumber(relative)
+---@param redraw boolean #whether to redraw the screen
+local function set_relativenumber(relative, redraw)
     local in_insert_mode = vim.api.nvim_get_mode().mode == "i"
 
     if vim.o.number then
         vim.opt.relativenumber = relative and not in_insert_mode
+        if redraw then
```
Comment (cpea2506) [inline lua/relative-toggle/init.lua:24]:
```suggestion

        if redraw then
```

A new line should be there for better readability

File: lua/relative-toggle/init.lua (hunk 1 above)
Comment (cpea2506) [reply lua/relative-toggle/init.lua:25]: It's okay. Currently, there is no lua API for `redraw` command

```suggestion
            vim.cmd "redraw"
```

**Comment by cpea2506** [comment]
Also, could you please fix up the typo of these two events in the `config.lua` file

- CmdLineEnter -> CmdlineEnter
- CmdLineLeave -> CmdlineLeave

https://github.com/cpea2506/relative-toggle.nvim/blob/e2a0d6f67b37f55fa4cf40c58b98beb53083dcf8/lua/relative-toggle/config.lua#L13-L14

**Comment by adam-ja** [comment]
> Thank you for detecting this earlier. I'm using `Noice.nvim`, they force `redraw` by default so I can't notice this.

No problem, thanks for the plugin! I've been meaning to switch to relative numbers for a while and this makes it more convenient.

I've made all the requested changes and also updated the readme to fix the event typos there.

**Review by cpea2506 — APPROVED** [review]
Thank you! Pretty much appreciate your work. 😁

**Pull request merged**